### PR TITLE
added ssl error troubleshooting

### PR DESCRIPTION
**Pull request by pawelpaszki**
## What
added ssl error troubleshooting

## Why
To allow running jenkins-jobs commands

## Verification Steps
Validate README.md update

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +50,13 @@ jenkins-jobs --conf /path/to/your/jenkins_jobs.ini update /path/to/your/template
...
 ```
+
+## Troubleshooting
+To prevent from getting SSL: CERTIFICATE_VERIFY_FAILED when running jenkins-jobs commands:
+`ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain`
+
+execute the following command in the terminal:
+
+```
+export PYTHONHTTPSVERIFY=0
```
Comment (matskiv) [inline README.md]: This is really a temporary solution. Have you tried installing Red Hat IT CA certificate?

Comment (pawelpaszki) [reply README.md]: installing certificate did not work for me

Comment (trepel) [reply README.md]: @pawelpaszki @matskiv I would vote for merging. I can't reproduce on my machine. I can't remember if I did some special setup/config of python to make it work. I don't have `PYTHONHTTPSVERIFY` defined.

Comment (pawelpaszki) [reply README.md]: I would probably merge it as well. It might not be the best solution, but it works for me

**Pull request merged**